tests/unit/commands/test_administration.py

```diff
@@ -0,0 +1,65 @@
+"""
+test_administration.py  - tests for the administration command suite
+
+Copyright (c) 2019 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+
+import logging
+
+from src.commands import administration
+from src.packages.context import Context
+
+LOG = logging.getLogger(f"mecha.{__name__}")
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_rehash_dm(context_pm_fx, bot_fx, async_callable_fx, monkeypatch):
+    """
+    Verifies rehash tells off the user when they attempt to invoke the command in private.
+    """
+
+    monkeypatch.setattr(administration, "setup", async_callable_fx)
+    async_callable_fx.return_value = ({}, '22aa')
+
+    await administration.cmd_rehash(context_pm_fx)
+
+    assert "everyone can see" in bot_fx.sent_messages[0]['message']
+    assert not async_callable_fx.was_called
+
+
+@pytest.mark.asyncio
+async def test_rehash_not_authorized(bot_fx, async_callable_fx, monkeypatch):
+    context = await Context.from_message(bot_fx, "#unittest", "some_recruit", "!rehash 👿")
```

Comment:
Hardcoded channel name?

Reply:
This is within a test file, im not sure i see the issue with using a hardcoded channel name?

The channel is ultimately irrelevant, it simply needs to be a valid channel name.

Further, in many test files we make use of similar hardcoded strings, why are you calling these out in specific?